otter/integration/tests/test_convergence.py

```diff
@@ -83,7 +83,6 @@ def test_scaling_to_clb_max_after_oob_delete_type1(self):
         for a CLB (as of this writing).
         """
         return self._perform_oobd_clb_test(25)
-    test_scaling_to_clb_max_after_oob_delete_type1.timeout = 1800
```

Comment:
Since they were all the same, there was an added class timeout so that the value is only set in one place. (That was done in a different PR and these timeouts were likely merged in parallel.)